src/explore-education-statistics-common/src/polyfill.js

```diff
@@ -42,13 +42,13 @@ if (!Element.prototype.matches) {
 }
 
 if (!Element.prototype.closest) {
-  Element.prototype.closest = s => {
+  Element.prototype.closest = function closest(s) {
```

Comment:
I know that this was in place before this PR so maybe outside its scope, but do we want to maintain our own polyfills?  Would it not be better to install polyfills with Lerna and then just import them where we need them?

Other than that, looks fine to me.

Reply:
It's a valid point, let's raise it in standup today